### ISSUE TITLE
-X (exclude columns) isn't working

### ISSUE THREAD

**Issue by dgilman** [schemaspy/schemaspy#474]
I'm using the latest 6.1 build of SchemaSpy and postgresql 10. If I run the jar with `-X ".*(foo|bar).*"` the output graphviz diagrams (relationships.real.compact, relationships.implied.large, relationships.implied.compact) still have the columns `foo` and `bar` in it, they're just shaded gray, which has the opposite effect of drawing attention to the columns instead of excluding them.

## Expected Behavior
-X should keep columns from appearing in graphviz output

## Current Behavior
Columns are shaded gray instead

**Comment by npetzall** [comment]
It might work as intended, documentation might be off.

Since we build of the old schemaspy 5.0 and from it's documenation
> -x "columnNamesRegex"
> Exclude matching columns from relationship analysis to simplify the generated graphs. This is a regular expression that's used to determine which columns to exclude. It must match table name, followed by a dot, followed by column name. For example: -x "(book.isbn) (borrower.address)" Note that each column name regular expression must be surround by ()'s and separated from other column names by a. Excluded relationships will still show up on detail pages.

>-X "columnNamesRegex"
> Same as -x but excluded relationships will not show up on detail pages.

If you want full exclusion that could be considered a feature request.

But I'll check the code and if it's only to exclude relationships in diagrams I'll update the documentation.

**Comment by MushyMiddle** [comment]
@npetzall 
I ran into this as well, and while actually excluding the columns from the diagrams completely would be nice, I can live with them being grayed-out for my purposes.  I would, however suggest simplifying the description of the Regex(es) above:

First, the Regex (it's really just one) is matched against "table-name.column-name".  It's a normal Java Pattern, so follows those semantics (https://docs.oracle.com/javase/8/docs/api/java/util/regex/Pattern.html).

Some examples:

Exclude the same column from all tables: ".*\.column-name"
Exclude multiple columns from all tables: ".*\.(column-name1|column-name2)"
Exclude multiple columns from multiple tables: "table1\.column-name1|table2\.column-name2"

(You can probably get away without the "\." and just use ".", but they are different in the world of Regex; and I'm a pedant)

I know it's beyond the scope of the docs to explain Regex's in detail, but a couple (valid) examples would help.  Technically, I don't think the examples you show would actually work, but the OP's example did/does.

All that said, thanks for your work on this very useful tool!

**Comment by npetzall** [comment]
@MushyMiddle yeah the `|` went missing at some point. The quoted text is from [Schemaspy 5.0 webpage](http://schemaspy.sourceforge.net/)

We should add your examples to [our doc](https://schemaspy.readthedocs.io/en/latest/configuration/commandline.html#diagram-related)
